spyder_line_profiler/tests/test_lineprofiler.py

```diff
@@ -17,7 +17,8 @@
 MAIN_APP = qapplication() 
```

Comment:
Could you remove this line and the import related with `qapplication`? I think that is what is causing the segfault on the Linux CI